packages/backend/server/src/core/features/service.ts

```diff
@@ -286,6 +287,57 @@ export class FeatureService {
       .then(r => r.count);
   }
 
+  async getWorkspaceConfig<F extends FeatureType>(
+    workspaceId: string,
+    feature: F
+  ): Promise<FeatureConfig<F> | undefined> {
+    const origFeature = await this.getFeature(feature);
+    const workspaceFeature = await this.prisma.workspaceFeature.findFirst({
+      where: {
+        workspaceId,
+        feature: { feature, type: FeatureKind.Feature },
+        activated: true,
+      },
+      select: { configs: true },
+    });
+
+    if (origFeature && workspaceFeature) {
+      const { name, configs } = origFeature;
+      const ret = FeatureConfigSchema.safeParse({
+        feature: name,
+        configs: Object.assign({}, configs, workspaceFeature.configs),
+      });
```

Comment:
this looks weird

Reply:
now the feature and config are mixed

Reply:
this config is intended to provide global switches at the workspace level but specific to team workspace, such as enableAi, which is indeed a bit weird
perhaps we can create a jsonb in the workspace and write the fields that can be added or removed into it (enableUrlPreview in the workspace table should also)

Reply:
I still prefer making simple and known boolean flags columns

Reply:
team workspace moved to quota, and enableXX moved to workspace table, configs is only related to quota now